app/enterprise/changelog.md

```diff
@@ -3,6 +3,89 @@ title: Kong Enterprise Changelog
 layout: changelog
 ---
 
+## 1.3.0.1
+**Release Date:** 2019/12/19
+
+### Features
+
+* Kong Enterprise now officially supports RHEL 8
+* (Alpha feature) Kong Enterprise can now perform encryption-at-rest for sensitive fields within the data store (Postgres or Cassandra).
+* (Alpha feature) Kong Enterprise now has a License Reports module for customers to view current usage metrics. For more information, contact your Kong Account Executive.
```

Comment:
```suggestion
```

Reply:
This needs to be in both 1.3 and 1.3.0.1, correct, @GayleNeumann ?